pkg/util/intlresource/match_test.go

```diff
@@ -35,12 +35,5 @@ func TestMatch(t *testing.T) {
 		})
 		So(err, ShouldBeNil)
 		So(matched, ShouldEqual, testItem("ja"))
-
-		// Match intl.DefaultLanguageTag if possible
-		matched, err = Match([]string{"zh"}, "ja", []LanguageItem{
-			testItem(intl.DefaultLanguage),
-		})
-		So(err, ShouldBeNil)
-		So(matched, ShouldEqual, testItem(intl.DefaultLanguage))
```

Comment:
We should remove `testItem(intl.DefaultLanguage)` in the other test case in the file to avoid confusion in future.